realtime/handlers.js

```diff
@@ -0,0 +1,8 @@
+let drive_collaboration_handlers = (socket, realtime) => {
+  socket.on("send-update", (update) => {
+    console.log(update);
+    socket.broadcast.emit("sync-update", update);
+  });
```

Comment:
Ah neat, maybe this should be scoped in to a room? 

![image](https://github.com/frappe/drive/assets/9079960/fc15f806-79b1-4832-8184-21258abef04c)

Reply:
Flow would be:
- user joins doc room
- user pushes changes to this event handler
- This event handler broadcasts update using doc room